astropy/wcs/tests/test_tab.py

```diff
@@ -46,7 +46,7 @@ def test_2d_spatial_tab_vs_model():
 
 
 @pytest.mark.skipif(
-    _WCSLIB_VER < Version("7.6"),
+    Version("7.6") > _WCSLIB_VER,
```

Comment:
How can we stop `ruff` from proposing this change?

Reply:
It's SIM300. The relative frequency of bad suggestions may be too high. We can 1) turn it off or 2) just turn off the autofix and annotate what we want to keep with ``# noqa: SIM300``.

Reply:
I think it is good to follow the convention that in an `assert` statement the value that is being tested is on the left and the prescribed value is on the right, but `SIM300` is making too many changes outside `assert` statements for my liking.

Reply:
Removed along with `SIM300`.